src/pybind/mgr/dashboard/controllers/ceph_users.py

```diff
@@ -70,13 +70,15 @@ def user_create(_, user_entity: str = '', capabilities: Optional[List[Cap]] = No
         return f"Successfully created user '{user_entity}'"
 
     @staticmethod
-    def user_delete(_, user_entity: str):
+    def user_delete(_, user_entities: str):
         """
         Delete a ceph user and it's defined capabilities.
```

Comment:
maybe add the documentation to clearly say that you can delete more than one user by specifying `,`? Or make it accept a list (but that would break backward compaitibility)

Reply:
I've updated the comments so it properly displays in the swagger now, what do you think?

<img width="886" height="380" alt="image" src="https://github.com/user-attachments/assets/54cd7347-3158-40f8-8d98-181824dba24e" />